### PR TITLE
UXG-1324 match design

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UXG-1324

### CODE REVIEW

**Comment by nicmarson** [comment]
axe-score
I ran axe before and after this PR, and there was a big improvement, 41 issues down to 14. The remaining 14 issues are from missing alt tags for the nav icons, and color contrast issues.

File: src/vue/backend/src/views/children/generalContentTab.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
             @form-field-update="formUpdate"
         ></listings-general>
         <idx-button
-            customClass="settings-button__save"
+            size="lg"
```
Comment (sgreenl5) [inline src/vue/backend/src/views/children/generalContentTab.vue:8]: The button sizes are not wide enough in all standalone pages, should be 155w

File: src/vue/backend/src/templates/GuidedSetupContentCard.vue [hunk 2]
```diff
@@ -84,7 +84,7 @@ export default {
     }
```
Comment (sgreenl5) [inline src/vue/backend/src/templates/GuidedSetupContentCard.vue:84]: The link button does not match design, in implementation it has blue color and a border/button color. In the design, it is black text

Comment (sgreenl5) [inline src/vue/backend/src/templates/GuidedSetupContentCard.vue:84]: Back button is not wide enough, should be 155px

**Review by sgreenl5 — CHANGES_REQUESTED** [review]

**Review by sgreenl5 — APPROVED** [review]

**Review by ErikIDX — APPROVED** [review]

**Pull request merged**